generate_demog_maps.py

```diff
@@ -0,0 +1,56 @@
+import argparse
```

Comment:
Maybe put this file in a folder e.g. `engagement_analysis`

Reply:
will move it in a future pr thanks.